llvm/lib/Target/Mips/MipsAsmPrinter.cpp

```diff
@@ -427,6 +477,13 @@ void MipsAsmPrinter::emitInstruction(const MachineInstr *MI) {
       emitBrsc(*OutStreamer, &*I);
       continue;
     }
+
+    if (Subtarget->hasNanoMips() && EmitNanoMipsBalcStubsReloc &&
+        I->getOpcode() == Mips::BALC_NM) {
+      if (emitDirectiveRelocBalcStubs(*I, OutContext, TM, *OutStreamer))
+        return;
```

Comment:
Just a style point: since the 5 conditionals above this are using 'continue', stick to that instead of returning.